### PR TITLE
Little change. More the same

### PR DESCRIPTION
**Pull request by Jalleo**
Removes a var and makes another one a FALSE and TRUE. That's it.

### CODE REVIEW

File: code/game/machinery/_machinery.dm [hunk 1]
```diff
@@ -119,10 +119,9 @@ Class Procs:
 	var/critical_machine = FALSE //If this machine is critical to station operation and should have the area be excempted from power failures.
 	var/list/occupant_typecache //if set, turned into typecache in Initialize, other wise, defaults to mob/living typecache
 	var/atom/movable/occupant = null
-	var/unsecuring_tool = /obj/item/wrench
```
Comment (Jalleo) [inline code/game/machinery/_machinery.dm]: literally nothing but one other location references this var. Why is it here?

File: code/game/machinery/_machinery.dm [hunk 2]
```diff
@@ -119,10 +119,9 @@ Class Procs:
...
 	var/interact_open = FALSE // Can the machine be interacted with when in maint/when the panel is open.
 	var/interact_offline = 0 // Can the machine be interacted with while de-powered.
-	var/speed_process = 0 // Process as fast as possible?
+	var/speed_process = FALSE // Process as fast as possible?
```
Comment (Jalleo) [inline code/game/machinery/_machinery.dm]: I believe I set this the right way around. Shout at me if I am wrong. Or just shout at me.

File: code/game/machinery/_machinery.dm (hunk 1 above)
Comment (ACCount12) [reply code/game/machinery/_machinery.dm]: It was here from the times new machinery construction/deconstruction was added, but it wasn't used.

File: code/game/machinery/_machinery.dm (hunk 2 above)
Comment (Supermichael777) [reply code/game/machinery/_machinery.dm]: yes FALSE = 0 is a define in our code and is the clear and preferred binary variable

**Review by ninjanomnom — APPROVED** [review]

**Comment by Jalleo** [comment]
Lets hope this doesn't become a merged at 20 hour mark.

**Comment by Cyberboss** [comment]
I love to prove people wrong

**Comment by Jalleo** [comment]
*sigh
People like to meme merge my simple changes it seems. Next time I will auto close it then open it after twenty four hours.......

**Pull request merged**